src/processor/reconnection-graph.service.ts

```diff
@@ -252,7 +256,7 @@ export class ReconnectionGraphService {
     }
 
     // If no pages to import, import at least one empty page so that user graph will be created
-    if (publicFollows.length + privateFollows.length + privateFriendships.length === 0) {
+    if (publicFollows.length + privateFollows.length + privateFriendships.length === 0 && (graphKeyPairs.length > 0 || dsnpKeys?.keys.length > 0)) {
```

Comment:
is this still necessary for updated graph sdk?

Reply:
it was a bug i believe since our first use case is private follows, this bug was hidden as keys were available from chain, in case i am a fresh dev provider and want to publish only public stuff, this breaks

Reply:
but its not a bug, ideally we should handle it here, graphsdk cannot do much about sending `undefined` `null` types because it will take it and fail to parse